modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/paths/ProcessBuilderListView.path

```diff
@@ -57,7 +57,7 @@
 </tr>
 <tr>
 	<td>CLEAR_BUTTON_PROCESS_BUILDER</td>
```

Comment:
This Path does the same thing as this other: ManagementBar#CLEAR_FILTER.
I think we could remove this CLEAR_BUTTON_PROCESS_BUILDER path, as it's only used in this failing test, and replace by this other path.

Reply:
Done